Dockerfile

```diff
@@ -11,9 +11,11 @@ RUN --mount=type=cache,sharing=locked,id=gomod,target=/go/pkg/mod/cache \
     --mount=type=cache,sharing=locked,id=goroot,target=/root/.cache/go-build \
     CGO_ENABLED=0 GOOS=linux make build
 
-FROM scratch
-# Add Certificates into the image, for anything that does API calls
-COPY --from=dev /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+FROM alpine:3.13
```

Comment:
Can you use alpine:3.15 here?

Reply:
Yes, fixed thank you.
FTR: I faced https://github.com/opencontainers/runc/issues/2151 locally; had to update docker to be able to work with alpine > 3.14 as stated here https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2